MyApplication/app/src/main/java/com/scottlindley/myapplication/MainActivity.java

```diff
@@ -0,0 +1,158 @@
+                        icon.setImageResource(R.drawable.runningback);
+                        break;
+                    case "tackle":
+                        icon.setImageResource(R.drawable.tackle);
+                        break;
+                    case "guard":
+                        icon.setImageResource(R.drawable.guard);
+                        break;
+                    case "wide receiver":
+                        icon.setImageResource(R.drawable.widereceiver);
+                        break;
+                    case "tight end":
+                        icon.setImageResource(R.drawable.tightend);
+                        break;
+                    case "center":
+                        icon.setImageResource(R.drawable.center);
+                        break;
+                }
+                return view;
+            }
+        };
+        mPlayerListView.setAdapter(mBaseAdapter);
+        mBaseAdapter.notifyDataSetChanged();
+    }
+
+    public void initButtons(){
+        mAddPlayerButton = (Button)findViewById(R.id.add_player_button);
+        mRemovePlayerButton = (Button)findViewById(R.id.remove_player_button);
+
+        mAddPlayerButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String editTextString = mEditPlayerName.getText().toString();
+                String positionString = mPositionDropDown.getSelectedItem().toString();
+                for(Player player : mPlayers){
+                    if(player.getName().equals(editTextString)
+                        &&player.getPosition().equals(positionString)){
+                        Toast.makeText(MainActivity.this, "Player already exists", Toast.LENGTH_SHORT).show();
+                        return;
+                    }
+                }
+                if (!(mEditPlayerName.getText().toString().equals(""))) {
+                    mPlayers.add(new Player(editTextString, positionString));
+                    mBaseAdapter.notifyDataSetChanged();
+                }else{
+                    mEditPlayerName.setError("Please enter player name");
+                }
+            }
+        });
+
+        mRemovePlayerButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                if(!mPlayers.isEmpty()){
+                    mPlayers.remove(mPlayers.size()-1);
+                    mBaseAdapter.notifyDataSetChanged();
+                }
+            }
+        });
+    }
+
+    public void initSpinner(){
+        mPositionDropDown = (Spinner)findViewById(R.id.position_drop_down);
+        ArrayList<String> positions = new ArrayList<>();
+        positions.add("Quarterback");
+        positions.add("Running Back");
+        positions.add("Tackle");
+        positions.add("Guard");
+        positions.add("Wide Receiver");
+        positions.add("Tight End");
+        positions.add("Center");
```

Comment:
Another option is to make a `string-array` in your strings.xml file containing all these positions as `item`s, then in your activity_main.xml you can add an attribute in the `Spinner` element for `android:entries` and specify that array with the value `@array/<whatever your array is named>`. Then you don't have to do this work in Java.